scripts/install-binaries

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+REV=a6a3290833f6da4a7785a426bed342df069ce47f
+
+BINARIES=(rad radicle-node radicle-httpd git-remote-rad)
+REPO_ROOT=$(git rev-parse --show-toplevel)
+BINARY_PATH=$REPO_ROOT/tests/tmp/bin/${REV:0:7}
+OS=$(uname)
+
+mkdir -p "$BINARY_PATH"
+
+for BINARY_NAME in "${BINARIES[@]}"
+do
+  # Handle different naming of binaries than required locally
+  if [ "$BINARY_NAME" = "rad" ]; then
+    DOWNLOAD_NAME=rad-debug
+    else
+    DOWNLOAD_NAME=$BINARY_NAME
+  fi
+
+  if ! [ -x "$(command -v "$BINARY_PATH/$BINARY_NAME")" ]; then
+    case "$OS" in
+      Darwin)
+        echo Downloading "$BINARY_NAME" from "https://files.radicle.xyz/$REV/aarch64-apple-darwin/$DOWNLOAD_NAME"
+        curl --fail -s "https://files.radicle.xyz/$REV/aarch64-apple-darwin/$DOWNLOAD_NAME" --output "$BINARY_PATH/$BINARY_NAME" || (echo "Download failed" && exit 1);;
+      Linux)
+        echo Downloading "$BINARY_NAME" from "https://files.radicle.xyz/$REV/x86_64-unknown-linux-musl/$DOWNLOAD_NAME"
+        curl --fail -s "https://files.radicle.xyz/$REV/x86_64-unknown-linux-musl/$DOWNLOAD_NAME" --output "$BINARY_PATH/$BINARY_NAME" || (echo "Download failed" && exit 1);;
+      *)       echo "There are no precompiled binaries for your OS: $OS, compile $BINARY_NAME manually and make sure it's in PATH." && exit 1 ;;
+    esac
+
+    chmod a+x "$BINARY_PATH/$BINARY_NAME"
+  fi
+done
+
+# Add binaries to corresponding path, depending on if CI or local
+if [ -n "$GITHUB_PATH" ]; then
+  echo "$BINARY_PATH" >> "$GITHUB_PATH"
+else
+  export PATH="$BINARY_PATH:$PATH"
+fi
```

Comment:
As discussed, we should just print the PATH here and offer the user instructions how to source it manually. This output could be used by the workflow to pass it to the next step.

Reply:
Nice! I would suggest the following edits to the output:
```
$ ./scripts/install-binaries

Using revision a6a3290833f6da4a7785a426bed342df069ce47f

Downloading rad from https://files.radicle.xyz/a6a3290833f6da4a7785a426bed342df069ce47f/aarch64-apple-darwin/rad-debug
Downloading radicle-node from https://files.radicle.xyz/a6a3290833f6da4a7785a426bed342df069ce47f/aarch64-apple-darwin/radicle-node
Downloading radicle-httpd from https://files.radicle.xyz/a6a3290833f6da4a7785a426bed342df069ce47f/aarch64-apple-darwin/radicle-httpd
Downloading git-remote-rad from https://files.radicle.xyz/a6a3290833f6da4a7785a426bed342df069ce47f/aarch64-apple-darwin/git-remote-rad

⚠️  You already have a radicle bin folder in your PATH variable.

To ensure you are using the correct binaries when running tests, run this command:
 👉 export PATH=/Users/rudolfs/work/radicle-interface/tests/tmp/bin/a6a3290:$PATH
```

- add a newline at the top and separate the warning message from the action with newlines
- print the full command that a user can just copy-paste

Reply:
I also think we should not print the warning if the correct binaries are in the path. Right now I get:

```
radicle-interface » which rad
/Users/rudolfs/work/radicle-interface/tests/tmp/bin/a6a3290/rad

radicle-interface » which radicle-node
/Users/rudolfs/work/radicle-interface/tests/tmp/bin/a6a3290/radicle-node

radicle-interface » which radicle-httpd
/Users/rudolfs/work/radicle-interface/tests/tmp/bin/a6a3290/radicle-httpd

radicle-interface » which git-remote-rad                                                                                                                                                                                   sebastinez/add-cli-tooling

/Users/rudolfs/work/radicle-interface/tests/tmp/bin/a6a3290/git-remote-rad

radicle-interface » ./scripts/install-binaries
Using revision a6a3290833f6da4a7785a426bed342df069ce47f

✅ rad
✅ radicle-node
✅ radicle-httpd
✅ git-remote-rad

⚠️  You already have a radicle bin folder in your PATH variable.
To ensure you are using the correct binaries make sure you add the following path to your PATH variable:

 👉 /Users/rudolfs/work/radicle-interface/tests/tmp/bin/a6a3290
```

Reply:
Ok think I found a good way with 9fc3807

Reply:
Which returns if `$BINARY_PATH` is found in PATH
```

Using revision a6a3290833f6da4a7785a426bed342df069ce47f

✅ rad
✅ radicle-node
✅ radicle-httpd
✅ git-remote-rad

✅ /Users/sebastinez/Projects/radicle-interface/tests/tmp/bin/a6a3290 is already in your PATH variable.
```